apps/www/_blog/2023-04-21-whats-new-in-pg-graphql-v1-2.mdx

```diff
@@ -15,18 +15,20 @@ thumb: launch-week-6/pggraphql/og-pg-graphql.png
 ---
 
 It's been 4 months since the [1.0.0 release of pg_graphql](https://supabase.com/blog/pg-graphql-v1).
-Since then, we’ve pushed several features to improve the APIs that `pg_graphql` produces. 
+Since then, we’ve pushed several features to improve the APIs that `pg_graphql` produces.
 
 In this article, we’ll walk through those features and show examples of each.
 
+<Admonition type="note">These features are only available after upgrading your project to v1.2.0. For help with upgrading, please review the documentation [here](https://supabase.com/docs/guides/platform/migrating-and-upgrading-projects).</Admonition>
```

Comment:
We don't currently have this component available on `www` (where the blog is hosted). Let's just stick to regular markdown for now.

Reply:
@gregnr 

I removed the component and replaced it with a div styling which was used here: `apps/www/_blog/2023-04-14-supabase-studio-2.0.mdx`

this is what it'll look like
<img width="400" alt="Screen Shot 2023-04-23 at 12 21 26 AM" src="https://user-images.githubusercontent.com/18966944/233819728-29c61669-f292-4704-ab40-5a80f2da7744.png">

let me know if you just want to remove the div entirely and have it as plain text instead.

Reply:
Looks great like this - thanks!